crates/tests/src/block.rs

```diff
@@ -1,8 +1,9 @@
 use core::cell::RefCell;
```

Comment:
Yes, they kind of are, but I simply wanted to put the effort in reproducing a maximum of them in order to show things indeed worked as expected and that no surprising difference with the normal API would occur. I can remove them if you want though. That can also be done once the normal API is upgraded to compute the string automatically.

Reply:
I'd prefer a few smoke tests, or if not that, then that the test file is made generic over the method (somehow) (because this is a lot of tests to maintain).